packages/config/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "@latticexyz/config",
+  "license": "MIT",
+  "version": "1.41.0",
+  "description": "Config for Store and World",
+  "type": "module",
+  "types": "dist/index.d.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/latticexyz/mud.git",
+    "directory": "packages/config"
+  },
+  "scripts": {
+    "prepare": "yarn build",
+    "test": "echo 'TODO tests'",
```

```suggestion
    "test": "vitest",
```
Could just use vitest (loooks like you were headed that way)

---

can't do that rn - I have no actual tests and vitest errors because of it, wouldn't want to break our CI

---

but yeah the plan is to do `vitest` when I make tests